storageconnector/pgtestsql.txt

```diff
@@ -0,0 +1,10 @@
+# simple sql querys to create a demo table for pazuzu in postgres
```

Comment:
2 things:
- We can already change extensions to `.sql`
- We have a folder called `storageconnector/fixtures` which fits much better for such files

My suggestion will be to call it something like: `storageconnector/fixtures/fixtures.sql`